src/prototype_room_init.js

```diff
@@ -375,6 +375,10 @@ Room.prototype.checkForSpawnPosition = function(pos) {
  * @return {undefined}
  **/
 Room.prototype.checkForMisplacedSpawn = function() {
+  const spawnsCount = Object.keys(Game.spawns).length;
```

Comment:
i dont think you should use Game.spawns as spawnsCount... check for 
```js
  const spawns = this.findMySpawns();
  if (spawns.length > 2) {
```

Reply:
@mschultheiss83 Done. please have a re-review.